nservicebus/upgrades/testing-6to7.md

```diff
@@ -0,0 +1,40 @@
+---
+title: NServiceBus Testing Upgrade Version 6 to 7
+summary: Instructions on how to upgrade NServiceBus.Testing Version 6 to 7.
+reviewed: 2017-09-28
+component: Testing
+related:
+ - nservicebus/testing
+ - nservicebus/upgrades/6to7
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 6
+ - 7
+---
+
+
+## Upgrade to NServiceBus Version 7
+
+NServiceBus.Testing requires NServiceBus Version 7.
+
+As part of upgrading to NServiceBus.Testing Version 7, projects will also require an upgrade to [NServiceBus Version 6](/nservicebus/upgrades/6to7/).
```

Comment:
Shouldn't the link text be `NServiceBus Version 7` and not `NServiceBus Version 6` ?